File: src/main/java/no/nav/service/pensjon/regler/to/BeregnOpptjeningResponse.java
package no.nav.service.pensjon.regler.to;

import java.util.ArrayList;
import java.util.List;

import no.nav.domain.pensjon.regler.Pakkseddel;
import no.nav.domain.pensjon.regler.grunnlag.Pensjonsbeholdning;

public class BeregnOpptjeningResponse extends ServiceResponse {

	private ArrayList<Pensjonsbeholdning> beholdninger; 
	
	/**
	 * Pakkseddel for svaret.
	 */
	private Pakkseddel pakkseddel = new Pakkseddel(true, true);

}

File: src/main/java/no/nav/pensjon/regler/to/BeregnOpptjeningResponse.kt
package no.nav.pensjon.regler.to

import no.nav.pensjon.regler.domain.Pakkseddel
import no.nav.pensjon.regler.domain.grunnlag.Pensjonsbeholdning
import java.io.Serializable

class BeregnOpptjeningResponse() : Serializable {
    var beholdninger: ArrayList<Pensjonsbeholdning> = ArrayList()

    var pakkseddel = Pakkseddel()

}